src/ui/components/MapBox/MapBox.js

```diff
@@ -0,0 +1,252 @@
+ui.class.MapBox = class MapBox extends HTMLElement {
+	constructor(props) {
+		super();
+		const self = this;
+
+		let m_divTag;
+		let m_map;
+		let m_parentTag;
+		let m_props;
+		let m_userLocation;
+		let m_zoomDefault = 0.01;
```

Comment:
All default values must be set in configure{}

Reply:
Done!